lib/exsoda/reader.ex

```diff
@@ -139,7 +139,7 @@ defmodule Exsoda.Reader do
       stream = "#{base}/id/#{Http.encode(state.fourfour)}.csv?#{query}"
       |> HTTPoison.get(Http.headers(state), [{:stream_to, self()} | options])
       |> as_line_stream
-      |> CSV.parse_stream(headers: false)
+      |> CSV.parse_stream(skip_headers: false)
```

Comment:
This is the only change required for the 0.1 → 1.0 change

Reply:
Wait, so are we skipping headers or not?

Reply:
Before, the parameter meant "there are no headers in the CSV file", now it means "don't skip any headers that might exist in the CSV file".  Confusing, definitely.

Reply:
1.0 docs:
> :skip_headers - when true, skips headers. Defaults to true. Set it to false to keep headers or when the CSV has no headers.

0.1 docs:
> :headers - if headers exist and, if so, they are discarded

Okay, so they do the same thing, it's just a really bad parameter name in 0.1. Cool.